client/src/pages/private/UserView.js

```diff
@@ -149,6 +149,7 @@ export default () => {
                 label="* User Role"
                 options={[
                   { value: 'health_authority', label: 'Health Authority' },
+                  { value: 'employer', label: 'Employer' },
```

Comment:
I wonder if we should use the label "Private Employer" here?

Reply:
Updated